Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabUndoHelpers.h

```diff
@@ -60,6 +60,17 @@ namespace AzToolsFramework
                 AZ::EntityId entityId,
                 UndoSystem::URSequencePoint* undoBatch);
 
+            //! Helper function for updating entities as overrides to focused template with undo-redo support.
+            //! @param entityList Entity list for entities to be updated in focused template.
+            //! @param owningInstance The common owning prefab instance of all inputs.
+            //! @param focusedInstance The current focused prefab instance.
+            //! @param undoBatch The undo batch node to register the update-entity undo node to.
+            void UpdateEntityListAsOverride(
```

Comment:
nit: override's' as there will most likely be one per each entity. Also 'UpdateEntitiesAsOverrides' might be a clearer name.